scripts/tests/chiptest/test_definition.py

```diff
@@ -107,6 +107,7 @@ def __startServer(self, runner, command, discriminator):
         logging.debug(
             'Executing application under test with discriminator %s.' % discriminator)
         app_cmd = command + ['--discriminator', str(discriminator)]
+        app_cmd = app_cmd + ['--interface-id', str(-1)]
```

Comment:
This new option stays optional right ? it s not mandatory ?

Reply:
Yes.